packages/boiler-task-selenium/src/runner/webdriverio/reporters/spec.js

```diff
@@ -0,0 +1,236 @@
+import events from 'events';
+
+
+/**
+ * Custom spec reporter adapted from https://github.com/Baltino/wdio-spec-reporter
+ *
+ */
+class SpecReporter extends events.EventEmitter {
+  constructor(baseReporter, config, options = {}) {
+    super();
+
+    this.baseReporter = baseReporter;
+    this.config = config;
+    this.options = options;
+
+    const { epilogue } = this.baseReporter;
```

Comment:
The only reporter that's included with the webdriverio package by default is the dot reporter.  We can also fork the existing spec reporter and point to that for the dependencies, but we really just need this one class until they resolve the parallel test issue.